src/main/scala/org/broadinstitute/dsde/agora/server/acls/RoleTranslator.scala

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.dsde.agora.server.acls
+
+import org.broadinstitute.dsde.agora.server.business.AgoraPermissions
+import org.broadinstitute.dsde.agora.server.business.AgoraPermissions._
+import org.broadinstitute.dsde.agora.server.acls.GcsRole._
+
+/**
+ * Created by dshiga on 6/16/15.
+ */
+object RoleTranslator {
+
+  def gcsBucketToNamespacePermissions(bucketRole: GcsBucketRole): AgoraPermissions = {
+    var namespacePermissions = new AgoraPermissions(AgoraPermissions.Nothing)
+    if (bucketRole.isReader) {
+      namespacePermissions = namespacePermissions.addPermissions(Read)
+    }
+    if (bucketRole.isWriter) {
+      namespacePermissions = namespacePermissions.addPermissions(Write)
```

Comment:
can't we say namespacePermissions.addPermission(Write, Create, Redact)?  I thought Jay implemented vararg constructor for AgoraPermissions